.github/workflows/build_and_test.yml

```diff
@@ -2,9 +2,11 @@ name: Build and test
 
 on:
   push:
-    branches: [ python-backend ]
-  pull_request:
-    branches: [ python-backend ]
+    paths-ignore:
+      - 'python/box.tests/reports/**'
+      - 'python/experiments/missing.txt'
```

Comment:
Content of `python/experiments` is now mixed: outputs, inputs and logic. How about having a designated directory for outputs? It will be more maintainable - we'll ignore the whole directory here and the risk of ignoring too much or too little will be reduced.

Reply:
Good idea, let me create the `generated` dir.